assets/css/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    background-color: #d9dcd6;
+}
```

Comment:
Maybe add a new line after this line just to create a separation and keep your code clean